rmgpy/molecule/group.py

```diff
@@ -1021,6 +1023,32 @@ def update(self):
         self.updateConnectivityValues()
         self.updateFingerprint()
 
+    def updateCharge(self):
+        """
+        Update the charge according to the valence_electron, total bond order, lone pairs
+        and radical electrons. When charged molecule is appearing in reactants, the product
+        template generated by applying recipe will result in wrong charge status. This
+        function is aimed to solve this problem
```

Comment:
I suggest to modify the wording a bit, e.g.: `Update partial charges according to the valence_electron, total bond order, lone pairs and radical electrons. This method is used for products of specific families with recipes that modify charges.`